Sources/StarscreamSwiftPhoenixClient/StarscreamTransport.swift

```diff
@@ -91,7 +91,7 @@ public class StarscreamTransport: NSObject, PhoenixTransport, WebSocketDelegate
   }
   
   public func send(data: Data) {
-    self.socket?.write(data: data)
+    self.socket?.write(string: String(data: data, encoding: .utf8)!)
```

Comment:
sending data frames makes phoenix use a [binary message](https://github.com/phoenixframework/phoenix/blob/a3a470e6bc4ecefbfac783f465e0fa130f672d4c/lib/phoenix/socket/serializers/v2_json_serializer.ex#L122-L141) format instead of the expected [text one](https://github.com/phoenixframework/phoenix/blob/a3a470e6bc4ecefbfac783f465e0fa130f672d4c/lib/phoenix/socket/serializers/v2_json_serializer.ex#L110-L120)

Reply:
v1 serializer is not affected by this change since it ignores opcode and decodes everything as if it's text